setup.py

```diff
@@ -71,7 +71,7 @@ def run(self):
     install_requires.append("networkx>=2.1")
 
 # Extra dependencies for remote integrations
-gs = ["google-cloud-storage==1.13.0"]
+gs = ["google-resumable-media==0.3.2", "google-cloud-storage==1.13.0"]
```

Comment:
Let's add a comment explaining why we need to explicitly fix google-resumable-media with a link to the issue, so we could understand this later.

Reply:
done

Reply:
done